.github/workflows/push_audit.yml

```diff
@@ -0,0 +1,21 @@
+name: Push Audit
+
+on: push
+
+jobs:
+  textformat:
+    name: EditorConfig audit
+    runs-on: ubuntu-latest
+    env:
+      ALWAYS_LINT_ALL_FILES: false
+      EC_FIX_ERROR: false
+    steps:
+      - name: CheckoutCode
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: EditorConfig-Action
+        uses: zbeekman/EditorConfig-Action@master
```

Comment:
I suggest the explanation in the commit message be entered here as a comment for easier maintenance / change to a more stable default in the future.

Reply:
Done, thx.